capa/version.py

```diff
@@ -1 +1,13 @@
 __version__ = "3.2.0"
+
+
+def get_major_version():
+    return int(__version__.partition(".")[0])
+
+
+def get_rules_branch():
+    return f"v{get_major_version()}"
+
+
+def get_rules_checkout_command():
+    return f"$ git clone https://github.com/mandiant/capa-rules.git -b {get_rules_branch()} /local/path/to/rules"
```

Comment:
Did you want to use this as additional output if rule parsing fails?

Reply:
looks great, thanks!